CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+[all-proposals]: https://github.com/hubotio/evolution/pulls?utf8=%E2%9C%93&q=is%3Apr
+[chat]: https://hubot-slackin.herokuapp.com/
+
+This document outlines how a Hubot feature grows from a rough idea into something that can improve workflows for millions of developers.
+
+This process covers all changes to Hubot, such as new features or adding, changing, or removing any APIs (no matter how small). Smaller changes, such as bug fixes, optimizations, or other improvements can be contributed via the [normal contribution process](https://github/github/hubot/blob/master/CONTRIBUTING.md) on the relevant repository.
+
+## Goals
+
+This process aims to leverage the collective ideas, insights, and experience of the community to improve the Hubot project and ecosystem. Its two primary goals are:
+
+* Engage the wider community in the ongoing evolution of Hubot, and
+* Maintain the vision and conceptual coherence of Hubot.
+
+There is a natural tension between these two goals. Open evolution processes are, by nature, chaotic. Yet, maintaining a coherent vision requires some level of coordination. This process aims to strike a balance that best serves the entire Hubot ecosystem.
+
+## Participation
+
+Everyone is welcome to propose, discuss, and review ideas to improve Hubot.
+
+The Hubot maintainers are responsible for the strategic direction of Hubot. Maintainers initiate, participate in, and manage the public review of proposals and have the authority to accept or reject changes to Hubot.
+
+## How to propose a change
+
+1. **Check prior proposals**: many ideas come up frequently, and may either be in active discussion, or may have been discussed and rejected. Please check the [list of proposals][all-proposals] before proposing something new.
+1. **Consider the goals of the upcoming release**: Each major release is focused on a [specific set of goals](README.md) described early in the release cycle. When proposing a change, please consider how your proposal fits in with the larger goals of the upcoming release. Proposals that are out of scope may still be considered, but will likely be postponed.
+1. **Socialize the idea**: Before starting the review process, it's helpful to gauge interest from the community. Ideally, the idea would have come up in a discussion in another issue or in [chat][] and the consensus was to start an evolution proposal.
+1. **Develop the proposal**: open a Pull Request with a rough sketch of the proposal using the [proposal template](template.md) in the [_drafts](https://github.com/hubotio/evolution/new/master/_drafts) directory. Prototyping an implementation and its uses along with the proposal is encouraged, because it helps ensure both technical feasibility of the proposal as well as validating that the proposal solves the problems it is meant to solve.
+1. **Request a review**: When you are ready for feedback from the Hubot community, change the frontmatter in your proposal from `status: in-progress` to `status: in-review`.
```

Comment:
Anyone have a better idea for how to manage this step? We need a way to announce to the community that a proposal is ready for review.

I was thinking we would add some automation here where changes to the frontmatter of the proposal would update the labels on the PR, cc @hubotio/maintainers, and post an announcement in Slack.